src/wavesurfer.ts

```diff
@@ -74,6 +74,8 @@ export type WaveSurferOptions = {
   fetchParams?: RequestInit
   /** Playback "backend" to use, defaults to MediaElement */
   backend?: 'WebAudio' | 'MediaElement'
+  /** Number of ms to debounce dragging when `dragToSeek` is true; defaults to 200ms */
+  dragDebounceTime?: number
```

Comment:
Should we maybe combine it with the `dragToSeek`? If false it should be disabled, if a number, it’s a debounce value and it’s enabled. It true, then 200.

Reply:
IMO that kind of overload could be confusing. Maybe if the type was `boolean | { debounceTime: number }`, but so far, that kind of layering doesn't match the rest of options.

The only other options that accept radically different types are the css passthroughs like `width?: number | string` but in that case I think it makes sense.

But it's your library, so I'll defer :) Either way is usable, and I'd love to use this feature!

Reply:
My thinking is that since drag-to-seek is already itself a togglable option, adding another dependent option that won’t do anything if dragToSeek isn’t enabled would also be somewhat confusing. Perhaps if it was called `dragToSeekDebounce` it would be more clear.
Your suggestion to make it a nested object makes sense to me tho, I would go with that.

Reply:
> "adding another dependent option that won’t do anything if dragToSeek isn’t enabled would also be somewhat confusing"

Very true, I didn't think of that. And with adding the nested object as a union to the boolean enable/disable type, it stays backwards compatible. I'll update.